src/server/auth/server/testing/oidc_test.go

```diff
@@ -70,12 +75,13 @@ func TestOIDCTrustedApp(t *testing.T) {
 // enterprise license is expired. We test this through the configured OIDC provider
 // and do the auth code flow.
 func TestCannotAuthenticateWithExpiredLicense(t *testing.T) {
-	if testing.Short() {
-		t.Skip("Skipping integration tests in short mode")
-	}
-	c, _ := minikubetestenv.AcquireCluster(t)
-	tu.ActivateAuthClient(t, c)
-	require.NoError(t, tu.ConfigureOIDCProvider(t, c))
+	t.Parallel()
+	env := realenv.NewRealEnv(t, dockertestenv.NewTestDBConfig(t))
+	env.ActivateIdentity("../../../../../dex-assets")
```

Comment:
since this is used in a bunch of places, could we ActivateIdentity with a functional option to NewRealEnv?

Reply:
So I tried to play around with this and unfortunately we can not. The best we can do is write an option to set one of the configuration variables to a magic value. Then if the value matches, we can call `ActivateIdentity()`. The underlying reason is that the config option allows us to change the `serviceenv.Configuration` struct, but we need to hook up changes to the realEnv as well. The solution I ended up going with for now is to define a new exported function for Identity being enabled.

Reply:
No problem. That makes sense. I'm giving this another pass and thinking through the existing code a bit more